src/foodmood/Account.java

```diff
@@ -0,0 +1,22 @@
+package foodmood;
+
+/**
+ *
+ * @author walte
+ */
+public class Account {
+    private String userName;
+    private String firstName;
+    private String lastName;
+    private String password;
+    
+    private Account(String userName1,String firstName1, String lastName1, String password1)
+    {
+        this.userName = userName1;
+        this.firstName = firstName1;
+        this.lastName = lastName1;
+        this.password = password1;
+    }
+    
```

Comment:
We'll need a method to authenticate the user by comparing a string parameter with the stored password.